sql/updates/2.8.0--2.8.1.sql

```diff
@@ -0,0 +1,180 @@
+
+-- gapfill with timezone support
+CREATE FUNCTION @extschema@.time_bucket_gapfill(bucket_width INTERVAL, ts TIMESTAMPTZ, timezone TEXT, start TIMESTAMPTZ=NULL, finish TIMESTAMPTZ=NULL) RETURNS TIMESTAMPTZ
+AS '@MODULE_PATHNAME@', 'ts_gapfill_timestamptz_timezone_bucket' LANGUAGE C VOLATILE PARALLEL SAFE;
+
```

Comment:
this is wrong the gapfill changes are not part of 2.8.1

Reply:
The migration has been removed.